### DISCUSSION TITLE
No logs, windows install.

### DISCUSSION THREAD

**Discussion by byte-opps** [apache/couchdb#3076] [Q&A]
Installed latest couchdb ver 3.1.0 to C:\CouchDB as per manual, windows installer on windows10.

added lines to conf 

[log]
file = ./couchdb.log

service restarted.

Also tryed full path C:\CouchDB\couchdb.log Service runs as system and this has can update/write to the config file in this path when :5984/_utils/#_config/couchdb@localhost web config page is updated.

service restarted.

Expected Behaviour = log file becreated and a logs written into this file.

**Comment by wohali** [comment]
Add

```
[log]
writer = file
``` 

to the file and restart the service.
